### DISCUSSION TITLE
Found Usage of Replit.com

### DISCUSSION THREAD

**Discussion by ghost** [serverless-dns/serverless-dns#63] [General]
Hi,
I recently found this Replit.com to try out using Deno and Node.js as a test using at main branch setup .
It was effective in testing out, it also have console,shell, env and other things that can be used.
It is also free to certain extent. ( check pricing).
I hope this maybe usefull for you if you have not heard it.

Thanks

**Comment by ignoramous** [comment]
You mean, we should consider adding support for deploys to replit?

**Reply by ghost** [reply]
Yes please, the more the better
